src/applications/hca/validation.js

```diff
@@ -56,13 +54,13 @@ export function validateMarriageDate(
   marriageDate,
   { spouseDateOfBirth, veteranDateOfBirth, discloseFinancialInformation },
 ) {
-  const vetDOB = moment(veteranDateOfBirth);
-  const spouseDOB = moment(spouseDateOfBirth);
-  const marriage = moment(marriageDate);
+  const vetDOB = Date.parse(veteranDateOfBirth);
+  const spouseDOB = Date.parse(spouseDateOfBirth);
+  const marriage = Date.parse(marriageDate);
```

Comment:
`marriageDate` is the second argument in `validateMarriageDate`.  All of these three arguments are ISO date strings.

https://github.com/department-of-veterans-affairs/vets-website/blob/8f57a94c2e86cc81b6d388d0af41c64af6286184/src/applications/hca/tests/validation.unit.spec.js#L93-L97